static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -287,11 +303,47 @@ function ReleaseContext(props: ReleaseContextProps) {
   }
 
   return (
-    <Fragment>
+    <Wrapper data-test-id="quick-context-hover-body">
       {renderReleaseDetails()}
       {renderIssueCountAndAuthors()}
       {renderLastCommit()}
-    </Fragment>
+    </Wrapper>
+  );
+}
+
+function EventContext(props: BaseContextProps) {
+  let stackTrace: StacktraceType | null = null;
+  const {isLoading, isError, data} = useQuery<Event>(
+    [
+      `/organizations/${props.organization.slug}/events/${props.dataRow['project.name']}:${props.dataRow.id}/`,
+    ],
+    undefined,
+    {
+      staleTime: fiveMinutesInMs,
+      retry: false,
```

Comment:
Should we really disable retries? I would only do this we expect this endpoint to return an error status under normal circumstances.

Reply:
Missed this, retries shudn't be disabled.